src/ui/controls/color-dropdown/style.less

```diff
@@ -0,0 +1,128 @@
+        background-image: @icon-down;
+        background-position: center right 0.25rem;
+        background-repeat: no-repeat;
+        background-size: 1rem;
+        border-radius: @dropdown-item-height / 2;
+        color: @color-control-fore;
+        display: inline-block;
+        height: @dropdown-item-height;
+        line-height: @dropdown-item-height;
+        margin-top: (@dropdown-height - @dropdown-item-height) / 2;
+        position: absolute;
+        width: 100%;
+        text-align: center;
+
+        &__two {
+            background-color: @color-control-active;
+            background-image: @icon-down;
+            background-position: center right 0.25rem;
+            background-repeat: no-repeat;
+            background-size: 1rem;
+            border-radius: @dropdown-item-height / 2;
+            color: @color-control-fore;
+            display: inline-block;
+            height: @dropdown-item-height;
+            line-height: @dropdown-item-height;
+            margin-top: (@dropdown-height - @dropdown-item-height) / 2;
+            position: absolute;
+            width: 100%;
+            text-align: unset;
+        }
+
+        &__text {
+            display: inline-block;
+            max-width: calc(100% - 2.5rem);
+            overflow: hidden;
+            text-overflow: ellipsis;
+            white-space: nowrap;
+        }
+    }
+
+    &__list {
+        background-color: @color-control-hover;
+        border-radius: @dropdown-item-height / 2;
+        box-shadow: 0 0 0.25rem black;
+        display: none;
+        margin-top: (@dropdown-height - @dropdown-item-height) / 2;
+        position: absolute;
+        width: 100%;
+
+        &__item {
+            color: @color-control-fore;
+            display: inline-block;
+            height: @dropdown-item-height;
+            line-height: @dropdown-item-height;
+            text-align: center;
+            width: 100%;
+            
+            &:hover {
+                background-color: @color-control-hover;
+                transition: background-color @time-fast;
+            }
+
+            &--selected {
+                overflow: hidden;
+                visibility: hidden;
+                white-space: nowrap;
+            }
+        }
+    }
+
+    &--open &__list {
+        border-radius: @dropdown-item-active-height / 2;
+        display: inline-block;
+        margin-top: (@dropdown-height - @dropdown-item-active-height) / 2;
+        z-index: 1;
+
+        &__item {
+            height: @dropdown-item-active-height;
+            line-height: @dropdown-item-active-height;
+        }
+    }
+
+    &--open &__selected {
+        border-radius: @dropdown-item-active-height / 2;
+        height: @dropdown-item-active-height;
+        line-height: @dropdown-item-active-height;
+        margin-top: (@dropdown-height - @dropdown-item-active-height) / 2;
+        z-index: 2;
+    }
+}
+
+.preview .color-dropdown {
+    cursor: pointer;
+}
+.color-div {
```

Comment:
Does this element make sense outside the ColorDropDown? If not, move it into `.color-dropdown`. It should have a classname like `.color-dropdown__reasonable-name`.